Mlem/App/Views/Shared/Palette Components/PaletteDivider.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ColoredDivider.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-08-30.
+//
+
+import Foundation
+import SwiftUI
+
+/// Divider() that colors itself appropriately to the palette.
+/// DO NOT use in UIKit environments!
+struct PaletteDivider: View {
```

Comment:
Do we want to name this `Divider` and override SwiftUI completely? We can always fallback to `SwiftUI.Divider` if we need to use the default for some reason. The same applies to `PaletteForm`

Reply:
Yeah, that seems cleaner.